11.hello-dapr-slim/README.md

```diff
@@ -0,0 +1,148 @@
+
+## Step 2 - Understand the Code
+
+Now that we've locally set up Dapr, clone the repo, then navigate to the Hello World sample: 
+
+```bash
+git clone https://github.com/dapr/samples.git
+cd samples/11.hello-dapr-slim
+```
+
+In the `app.js` you'll find a simple `express` application, which exposes a single route and handler. First, let's take a look at the top of the file: 
+
+Let's take a look at the ```neworder``` handler:
+
+```js
+app.post('/neworder', bodyParser.json(), (req, res) => {
+    const data = req.body.data;
+    const orderId = data.orderId;
+    console.log("Got a new order! Order ID: " + orderId);
+    res.status(200).send("Got a new order! Order ID: " + orderId);
+});
+```
+
+Here we're exposing an endpoint that will receive and handle `neworder` messages. We log the incoming message, and respond with the same log.
+
+## Step 3 - Run the Node.js App with Dapr
+
+1. Install dependencies: 
+
+    ```sh
+    npm install
+    ```
+
+    This will install `express` and `body-parser`, dependencies that are shown in our `package.json`.
+
+2. Run Node.js app with Dapr: 
+
+    ```sh
+    dapr run --app-id nodeapp --app-port 3000 --port 3500 node app.js
+    ```
+
+The command should output text that looks like the following, along with logs:
+
+```
+Starting Dapr with id nodeapp. HTTP Port: 3500. gRPC Port: 9165
+You're up and running! Both Dapr and your app logs will appear here.
+...
+```
+> **Note**: the `--app-port` (the port the app runs on) is configurable. Our Node app happens to run on port 3000, but we could configure it to run on any other port. Also note that the Dapr `--port` parameter is optional, and if not supplied, a random available port is used.
+
+## Step 4 - Post Messages to your Service
+
+Now that Dapr and our Node.js app are running, let's POST messages against it, using different tools. **Note**: here we're POSTing against port 3500 - if you used a different port, be sure to update your URL accordingly.
+
+First, let's POST the message by using Dapr cli in a new command line terminal:
+
+Windows Command Prompt
+```sh
+dapr invoke --app-id nodeapp --method neworder --payload "{\"data\": { \"orderId\": \"41\" } }"
+```
+
+Windows PowerShell
+```sh
+dapr invoke --app-id nodeapp --method neworder --payload '{\"data\": { \"orderId\": \"41\" } }'
+```
+
+Linux or MacOS
+```sh
+dapr invoke --app-id nodeapp --method neworder --payload '{"data": { "orderId": "41" } }'
+```
+
+Now, we can also do this using `curl` with:
+
+```sh
+curl -XPOST -d @sample.json -H "Content-Type:application/json" http://localhost:3500/v1.0/invoke/nodeapp/method/neworder
+```
+
+Or, we can also do this using the Visual Studio Code [Rest Client Plugin](https://marketplace.visualstudio.com/items?itemName=humao.rest-client)
+
+[sample.http](sample.http)
+```http
+POST http://localhost:3500/v1.0/invoke/nodeapp/method/neworder
+
+{
+  "data": {
+    "orderId": "42"
+  } 
+}
+```
+
+Last but not least, we can use the Postman GUI.
```

Comment:
"Last but not least" is not the tone we usually have in the docs, please change. Also reference to "we"